.github/workflows/pull-request-management.yml

```
@@ -71,8 +71,23 @@ jobs:
     needs: file-changes
     if: needs.file-changes.outputs.dhcp_module == 'true' || needs.file-changes.outputs.code_v3 == 'true' || needs.file-changes.outputs.code_v1 == 'true'
     steps:
+      # Removed as gh action is marked as deprecated
+      # - uses: pre-commit/action@v2.0.2
+      - name: 'set environment variables'
+        run: |
+          echo "PY_COLORS=1" >> $GITHUB_ENV
```

According to [this line](https://github.com/pre-commit/pre-commit/blob/master/pre_commit/color.py#L103) or [this doc](https://pre-commit.com/#command-line-interface) PY_COLORS is not consumed by pre-commit (like it is by molecule or pytest). You should use the following instead:
```suggestion
          echo "PRE_COMMIT_COLOR =always" >> $GITHUB_ENV
```

---

You should also use `--color always` argument to pre-commit command. Actually this would be better since you will have one less step in your job.